### PR TITLE
Add lftp

### PR DESCRIPTION
**Pull request by dzogrim**
This PR is about add support for lftp.rc ; thanks.

### CODE REVIEW

File: mackup/applications/lftp.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[application]
+name = lftp
+
+[configuration_files]
+.config/lftp/rc
```
Comment (lra) [inline mackup/applications/lftp.cfg]: You should leverage the XDG config folder, see https://github.com/lra/mackup/blob/master/mackup/applications/tig.cfg#L7-L8 for an example.

Comment (dzogrim) [reply mackup/applications/lftp.cfg]: OK! thanks @lra I made modifications as suggested…

Comment (dzogrim) [reply mackup/applications/lftp.cfg]: 👍

File: mackup/applications/lftp.cfg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[application]
+name = lftp
+
+[configuration_files]
+rc
```
Comment (lra) [inline mackup/applications/lftp.cfg]: What's this one?

File: mackup/applications/lftp.cfg [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+[xdg_configuration_files]
+.config/lftp
```
Comment (lra) [inline mackup/applications/lftp.cfg]: You don't have to specify the XDG folder, I suggest trying your conf until it works before opening a PR.

Thanks!

**Pull request closed without merging**